### PR TITLE
New CICE Grid for 01deg ryf

### PR DESCRIPTION
**Pull request by anton-seaice**
Contributes to #92

### CODE REVIEW

**Review by aidanheerdegen — DISMISSED** [review]
LGTM

**Comment by aidanheerdegen** [comment]
@CodeGat pointed out this was a PR directly to `release`. Should be `dev-`, have changed the base and will re-review.

**Comment by anton-seaice** [comment]
Thankyou. I knew i would get one wrong !

File: metadata.yaml [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
-  Year Forcing strategy outlined by Stewart et al. (2020), 
-  https://doi.org/10.1016/j.ocemod.2019.101557.
-  The configuration is based on that described in Kiss et al. (2020),
-  https://doi.org/10.5194/gmd-13-401-2020, but with many improvements.
-  Initial conditions are WOA13v2 potential temperature and practical salinity.
-  Run with JRA55-do v1.4.0 RYF9091 (1 May 1990 - 30 April 1991) repeated forcing
-  with all solid runoff converted to liquid runoff with no heat transfer.
-  Spin up starts from a nominal year of 1 Jan 1900.
+description: "0.1 degree ACCESS-OM2 global model configuration under the RYF9091 Repeat\nYear Forcing strategy outlined by Stewart et al. (2020), \nhttps://doi.org/10.1016/j.ocemod.2019.101557.\nThe configuration is based on that described in Kiss et al. (2020),\nhttps://doi.org/10.5194/gmd-13-401-2020, but with many improvements.\nInitial conditions are WOA13v2 potential temperature and practical salinity.\nRun with JRA55-do v1.4.0 RYF9091 (1 May 1990 - 30 April 1991) repeated forcing\nwith all solid runoff converted to liquid runoff with no heat transfer.\nSpin up starts from a nominal year of 1 Jan 1900."
```
Comment (aidanheerdegen) [inline metadata.yaml:4]: It seems that the GitHub bot commit did a bad thing and replaced newlines with `\n` characters. We don'y know why it did this, and it isn't your fault, but could you fix it in this PR?

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
Could you please make the change indicated? Thanks!

**Comment by aidanheerdegen** [comment]
> Thankyou. I knew i would get one wrong !

Actually it was really helpful, as it highlighted a logic error in the CI which should have refused to allow the PR (we only want PRs into `release-` branches to be able to come from the corresponding `dev-` branch).

File: metadata.yaml (hunk 1 above)
Comment (anton-seaice) [reply metadata.yaml]: Ok done :)

**Review by aidanheerdegen — APPROVED** [review]
Thanks!

**Pull request merged**